pastas/plots.py

```diff
@@ -1017,18 +1017,20 @@ def series(head=None, stresses=None, hist=True, kde=False, titles=True,
                       grid=False, density=True)
         if kde:
             gkde = gaussian_kde(head, bw_method='scott')
-            sample_range = np.nanmax(head) - np.nanmin(head)
-            ind = np.linspace(np.nanmin(head) - 0.1 * sample_range,
-                              np.nanmax(head) + 0.1 * sample_range, 1000)
+            sample_range = np.max(head) - np.min(head)
+            ind = np.linspace(np.min(head) - 0.1 * sample_range,
+                              np.max(head) + 0.1 * sample_range, 1000)
             if hist:
-                colour = 'C5'
+                colour = 'C1'
             else:
                 colour = 'k'
             axes[0, 1].plot(gkde.evaluate(ind), ind, color=colour)
         if hist or kde:
             # stats table
             head_stats = [["Count", f"{head.count():0.0f}"],
                           ["Mean", f"{head.mean():0.2f}"],
+                          ["Max", f"{head.max():0.2f}"],
+                          ["Min", f"{head.min():0.2f}"],
```

Comment:
Where f-strings faster than "{:0.2f}".format here?

Reply:
I don't know. I always use this way of formatting strings. 

Found this on https://blog.finxter.com/string-formatting-vs-format-vs-formatted-string-literal/:
> Summary: f-string is more readable and easier to implement than % and .format() string formatting styles. Furthermore, using f-strings is suggested for Python 3.6 and above while .format() is best suited for Python 2.6 and above. Versions prior to Python 2.6 only provide % option for string formatting. In terms of speed, % is the fastest string formatting style. 

I can change it if you want.